samples/csharp/getting-started/Ranking_AutoML/Ranking/Ranking/Ranking.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+	<ItemGroup>
+		<Compile Include="..\..\..\..\common\AutoML\ConsoleHelper.cs" Link="Common\ConsoleHelper.cs" />
+		<Compile Include="..\..\..\..\common\AutoML\ProgressHandlers.cs" Link="Common\ProgressHandlers.cs" />
+	</ItemGroup>
+
+	<ItemGroup>
+    <PackageReference Include="Microsoft.ML.AutoML" Version="0.17.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Assets\" />
+    <Folder Include="MLModel\" />
+  </ItemGroup>
+
+</Project>
```

Comment:
Can also benefit from a README.md for the sample, and link from the repo's main [README.md](https://github.com/dotnet/machinelearning-samples/blob/master/README.md#automl-api-samples-preview-state).

For the sample's README.md, I think the main work would be modifying the [existing ranking readme](https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/Ranking_Web) by replacing in the AutoML code.

_(tagged on a line of code to allow for a threaded discussion)_

Reply:
README file was now added. Good suggestion!